pkg/util/httpstream/spdy/roundtripper_test.go

```diff
@@ -154,7 +204,24 @@ func TestRoundTripAndNewConnection(t *testing.T) {
 			t.Fatalf("%s: Error creating request: %s", k, err)
 		}
 
-		spdyTransport := NewRoundTripper(testCase.clientTLS)
+		spdyTransport := NewSpdyRoundTripper(testCase.clientTLS)
+
+		var proxierCalled bool
+		if testCase.withProxy {
+			proxyHandler := goproxy.NewProxyHttpServer()
```

Comment:
add a hook to the proxyHandler to record when the proxy server is called to make sure we actually hit it